apps/web/src/client/components/accessible-autocomplete.js

```diff
@@ -0,0 +1,12 @@
+import 'accessible-autocomplete';
+
+// on page load, progressively enhance all <select> elements with the
+// "accessible-autocomplete" class
+window.addEventListener('DOMContentLoaded', function () {
```

Comment:
We defer the script so there is no need for this. Also I think I will preload the JS and I will move it at the bottom.